internal/xdscache/v2/endpointstranslator.go

```diff
@@ -272,11 +272,11 @@ func (e *EndpointsTranslator) OnChange(d *dag.DAG) {
 			if err := svc.Validate(); err != nil {
 				e.WithError(err).Errorf("dropping invalid service cluster %q", svc.ClusterName)
 			} else if _, ok := names[svc.ClusterName]; ok {
-				e.Errorf("dropping service cluster with duplicate name %q", svc.ClusterName)
+				e.Debugf("dropping service cluster with duplicate name %q", svc.ClusterName)
```

Comment:
Maybe `Warnf` here, since we don't really expect there to be duplicate service clusters? If the names collide, we don't know whether that's the same thing multiple times or different things with the name name.

Reply:
I guess that's fine. It is unfortunate that it will generate Warn logs for valid situations, though, e.g. when two routes use the same service, so the service gets visited twice.

Reply:
Yeh, maybe debug is better then

Reply:
I put it at debug; if anyone else has strong feelings LMK.